application-admintools-default/src/main/java/com/xwiki/admintools/internal/data/identifiers/TomcatIdentifier.java

```diff
@@ -82,6 +89,15 @@ public void updatePossiblePaths()
             String.format("%s/webapps/xwiki/WEB-INF/", this.serverPath) };
     }
 
+    @Override
+    public Map<String, String> getServerMetadata()
+    {
+        ServletContainerPing servletPing = pingProvider.getServletPing();
+        String serverName = servletPing.getName();
+        String serverVersion = servletPing.getVersion();
+        return Map.of("serverName", serverName, "serverVersion", serverVersion);
```

Comment:
Since this is a map of server metadata, maybe you can use only `name` and `version`?